dashboard/config/locales/en.yml

```diff
@@ -1404,9 +1404,25 @@ en:
     founder: "Founder, Code.org"
     your_friends: "Your Friends at Code.org"
   lti:
-    create_integration_success: "Succesfully created your LTI integration"
+    create_integration_success: "Successfully created your LTI integration"
     error:
       integration_exists: "Lti Integration already exists for this Client ID"
-      missing_params: "Missing required param(s): client_id, lms, email"
+      missing_params: "Missing required param(s): Client ID, LMS, Email"
       unsupported_lms_type: "Unsupported LMS platform type"
       wrong_resource_type: "Only LtiResourceLink is supported right now"
+    integration_client_id: "LMS Client ID"
```

Comment:
The links contained in some of these strings will be updated in a followup PR when Ali provides them.